### ISSUE TITLE
Unable to import .NEF generated by Nikon CoolScan IV ED

### ISSUE THREAD

**Issue by kirklandarchitect** [darktable-org/darktable#7742]
NEF images generated by Nikon CoolScan IV ED throw the following error on import (and show the skull): 

"[rawspeed] (TEST.nef) virtual std::unique_ptr<rawspeed::RawDecoder> rawspeed::RawParser::getDecoder(const rawspeed::CameraMetaData*), line 91: No decoder found. Sorry."

NEF images generated by Nikon D-850 work fine. 
NEF images generated by Nikon CoolScan IV ED work fine with Windows Photo Viewer and any Nikon viewer (ViewNX-i, etc.). 

To reproduce: download a sample CoolScan NEF image (https://1drv.ms/u/s!Ag9FccAVark6341RfudWfblnDLWCPw?e=TIkqRv) and select [import] > [image...].

Notes: 
1) Comparing the "image information" the only perhaps relevant differences between CoolScan NEF and Nikon camera NEF appear to be the "flags", "lens" and "datetime" attributes;
2) CoolScan NEF images show fine in the small preview area to the right of the "import image" dialog.

Thanks for your attention. If I have to post the issue elsewhere please let me know. 

Platform:
 - Darktable Version: 3.4.0
 - OS: Windows 10 Pro (Ver: 20H2, OS Build: 19042.685)
 - OpenCL: No

**Comment by johnny-bit** [comment]
Please follow https://github.com/darktable-org/darktable/wiki/Camera-support especially make sure you've sent a sample file to raw.pixls.us and please open a new issue using [new camera support issue](https://github.com/darktable-org/darktable/issues/new?assignees=LebedevRI&labels=scope%3A+camera+support&template=new_camera_support.md&title=%3CEnter+your+camera+name+here%3E)

**Comment by kmilos** [comment]
@kirklandarchitect Please see https://github.com/darktable-org/darktable/issues/7649 for an an example of how the process works.

**Comment by kirklandarchitect** [comment]
Thank you,

according to the page at raw.pixls.us I have successfully uploaded a sample .NEF file (35.9MB NEF) but doesn’t show up in the repository yet. Any delay perhaps?

In any case my issue with Nikon CoolScan IV ED (and same Nikon Scan 4.0.3 software) looks identical to NIKON Coolscan V - Nikon Scan 4.0.3 · Issue #7649 · darktable-org/darktable (github.com)<https://github.com/darktable-org/darktable/issues/7649> posted just a few days ago.

**Comment by johnny-bit** [comment]
> according to the page at raw.pixls.us I have successfully uploaded a sample .NEF file (35.9MB NEF) but doesn’t show up in the repository yet. Any delay perhaps?

All files have to be manually checked by maintainers of raw.pixls.us project.

Also I remember one thing - possibly if your scanner allows you to choose size, try to upload smaller size too?

**Comment by kirklandarchitect** [comment]
Thanks for the clarification.

Unfortunately the Nikon Scan software (which as far as I know is the only way to save the film scan produced by CoolScan devices) doesn’t offer image quality options for NEF (or TIFF), only for JPEG.  So for NEF it‘s always about 35MB each scan.

**Comment by kmilos** [comment]
I think it's **dpi** you're after in order to reduce the total number of pixels in the image, not quality settings like compression...

E.g. the other user of CoolScan V managed to save a very small [image w/ 600 dpi setting](https://raw.pixls.us/getfile.php/4269/exif/Image21-600pixels.nef.exif.txt).

**Comment by kirklandarchitect** [comment]
Thanks,

just uploaded a 1.6MB NEF scanned at 600dpi by CoolScan IV Ed and captured by Nikon Scan 4.0.3.

At this point my entry #7742 looks very much a duplicate of #7649: different scanner model, but identical .NEF generating software and outcome.

Cheers

**Comment by kmilos** [comment]
Yes, it does look like it's probably only the SW that matters when it comes to scanner files...
